### ISSUE TITLE
Change in service.dead behavior in Windows

### ISSUE THREAD

**Issue by sbreidba** [saltstack/salt#37740]
### Description of Issue/Question

The `service.dead` state behaves differently between versions 2016.3.3 and 2016.3.4. In 2016.3.3, executing `service.dead` on a service that is not yet installed succeeds. However, in 2016.3.4, it fails.

This is a problem for me because my sequence is:

1. Use the graceful-service-down example which uses `service.dead` and a prereq requisite on a file state to check if files need to be copied for the service.
1. Because 'prereq' is used, this stops the service if files will be changed, otherwise it does not.
1. Files are copied.
1. The service is installed (I use a custom state & module for this step that invokes InstallUtil).

For a fresh installation this works fine in 2016.3.3 and earlier. The `service.dead` state runs the file copy in test mode, determines that changes need to be made, stops the (non-existent) service successfully, and executes the file copy. The service install then succeeds. 

However, in 2016.3.4, the files are not copied, because `service.dead` fails, causing the service install to fail (since nothing was copied).

Again, this based on the `graceful-down` example in https://docs.saltstack.com/en/latest/ref/states/requisites.html. Is there a better approach? For now I have replaced a direct call of `service.dead` with an indirect call to it via `module.run`, but this will no longer fail if the service did exist and could not be stopped, which I find to be sub-optimal.

### Setup

A state file with something like:
```
stop-my-service:
  service.dead:
    - name: my-non-existent-service
```

Note that this state executes with `Result: True` in 2016.3.3, but `Result: False` in 2016.3.4.

### Versions Report
> c:\salt\salt-call --local --versions-report
```
Salt Version:
           Salt: 2016.3.4

Dependency Versions:
           cffi: 1.6.0
       cherrypy: 5.3.0
       dateutil: 2.5.3
          gitdb: 0.6.4
      gitpython: 2.0.2
          ioflo: 1.5.3
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: 1.4.5
       M2Crypto: Not Installed
           Mako: 1.0.4
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.12 (v2.7.12:d33e0cf91556, Jun 27 2016, 15:19:22) [MSC v.1500 32 bit (Intel)]
   python-gnupg: 0.3.8
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: Not Installed
          smmap: 0.9.0
        timelib: 0.2.4
        Tornado: 4.3
            ZMQ: 4.1.2

System Versions:
           dist:
        machine: AMD64
        release: 7
         system: Windows
        version: 7 6.1.7601 SP1 Multiprocessor Free
```

**Comment by Ch3LL** [comment]
@sbreidba this looks to be a duplicate of #37511 which is fixed in #37562

Want to give that fix a try and see if it resolves the issue?

**Comment by sbreidba** [comment]
Apologies for the duplicate. I'll try this fix and track any further comments in #37562. Closing.
